nx/include/switch/services/fs.h

```diff
@@ -77,6 +77,18 @@ typedef struct
     u64 unk_x38;                    ///< 0 for SystemSaveData/SaveData.
 } FsSave;
 
+/// SaveCreate Struct
+typedef struct {
+    u64 size;           ///< Size of the save data.
+    u64 journalSize;    ///< Journal size of the save data.
+    u64 blockSize;      ///< Block size of the save data.
+    u64 ownerId;        ///< Id of the save data's owner.
```

Comment:
Clarify what exactly the Id is?(titleID?)

Reply:
Yeah it's a title id. Done.